uint8 camera
uint8 relay
uint8 ur5_state
